operator/cmd/cilium_node.go

```diff
@@ -313,39 +314,41 @@ func processNextWorkItem(queue workqueue.RateLimitingInterface, syncHandler func
 	return true
 }
 
-type ciliumNodeUpdateImplementation struct{}
+type ciliumNodeUpdateImplementation struct {
+	clientset k8sClient.Clientset
```

Comment:
Now that the type `ciliumNodeUpdateImplementation` has the `clientset` field, we should initialize it properly when creating an instance [here](https://github.com/joamaki/cilium/blob/22ac15ff735f396bcbbd4e3d1b8759c012389031/operator/cmd/root.go#L394).

Reply:
Yep this was mostly what I referred to with "minor issues" (it was cause for failing tests). Fixed.